Makefile

```diff
@@ -1,8 +1,12 @@
 .PHONY: clean-pyc clean-build docs clean
 
-VIRTUALENV = virtualenv --python=python3
-PYTHON = $(VENV)/bin/python
-VENV := $(shell echo $${VIRTUAL_ENV-.venv})
+VENV = venv
+PYTHON=$(VENV)/bin/python
+
+# make it work on windows too
+ifeq ($(OS), Windows_NT)
+    PYTHON=$(VENV)/Scripts/python
+endif
```

Comment:
hey @gauravp446 CI pipeline is failing from your changes, do this
```
VENV = venv
PYTHON=$(VENV)/bin/python
```

Add these two lines in the `ifeq` block and revert back these 3 lines, this should fix the issue.
```
VIRTUALENV = virtualenv --python=python3
PYTHON = $(VENV)/bin/python
VENV := $(shell echo $${VIRTUAL_ENV-.venv})
```

Reply:
Hi @sumit4613 within the `ifeq` block it should be :
`
VENV = venv
PYTHON=$(VENV)/bin/python
`
?